library/src/androidTest/java/com/google/maps/android/data/kml/KmlStyleTest.java

```diff
@@ -81,9 +114,54 @@ public void testLineColor() {
     }
 
     @Test
-    public void testMarkerColor() {
+    public void testMarkerColorLeadingSpace() {
+        if (TestUtil.isRunningOnTravis()) {
+            Assume.assumeTrue("Skipping KmlStyleTest.testMarkerColorLeadingSpace() - this is expected behavior on Travis CI (#573)", false);
+            return;
+        }
         KmlStyle kmlStyle = new KmlStyle();
         assertNotNull(kmlStyle);
         assertNotNull(kmlStyle.getMarkerOptions());
+        kmlStyle.setMarkerColor(" FFFFFF");
+        // We can't get the marker color to see if it set correctly
```

Comment:
How about updating the visibility of [KmlStyle.mMarkerColor](https://github.com/googlemaps/android-maps-utils/blob/master/library/src/main/java/com/google/maps/android/data/kml/KmlStyle.java#L46) and annotate it with [VisibleForTesting](https://developer.android.com/reference/androidx/annotation/VisibleForTesting)? Otherwise, without inspecting the resulting marker color this test and `testMarkerColorTrailingSpace` should be removed.

Reply:
Sounds good - I just changed this in https://github.com/googlemaps/android-maps-utils/pull/599/commits/82f131d93a523fdd3f252c96dc5903a30fc33515. FWIW the existing unit test was still ensuring that an exception wasn't thrown, but I agree this is a better solution.